src/sqlfluff/dialects/dialect_tsql.py

```diff
@@ -1392,6 +1392,14 @@ class TableOptionSegment(BaseSegment):
                                     OneOf("ON", "OFF"),
                                     optional=True,
                                 ),
+                                Sequence(
```

Comment:
Could `HISTORY_TABLE`, `DATA_CONSISTENCY_CHECK`, and `HISTORY_RETENTION_PERIOD` be given in any order? If so, I would put them in a `Delimited` which handles this and the `CommaSegment`

Reply:
Yes it can, I'll correct this.